workers/unity/Packages/com.improbable.gdk.tools/DownloadCoreSdk.cs

```diff
@@ -29,8 +49,7 @@ internal static class DownloadCoreSdk
         {
             PluginDirectoryCompatibility.CreateWithCompatiblePlatforms("Assets/Plugins/Improbable/Core/OSX", new List<BuildTarget> { BuildTarget.StandaloneOSX }, editorCompatible: true),
             PluginDirectoryCompatibility.CreateWithCompatiblePlatforms("Assets/Plugins/Improbable/Core/Linux", new List<BuildTarget> { BuildTarget.StandaloneLinux64 }, editorCompatible: true),
-            PluginDirectoryCompatibility.CreateWithCompatiblePlatforms("Assets/Plugins/Improbable/Core/Windows/x86_64", new List<BuildTarget> { BuildTarget.StandaloneWindows64 }, editorCompatible: true),
-            PluginDirectoryCompatibility.CreateWithCompatiblePlatforms("Assets/Plugins/Improbable/Core/Windows/x86", new List<BuildTarget> { BuildTarget.StandaloneWindows }, false),
+            PluginDirectoryCompatibility.CreateWithCompatiblePlatforms("Assets/Plugins/Improbable/Core/Windows", new List<BuildTarget> { BuildTarget.StandaloneWindows64 }, editorCompatible: true),
```

Comment:
x86 directory was throwing warnings, was not consistent with Linux one and we don't actually have any Windows x86 plugins

Reply:
that is not correct. we are simply not downloading it right now, however, we do have x86 plugins that could be used

I fixed the linux one last night. please run cloud deployments and test everything before merging in PRs

Reply:
Yeah, saw that, thanks, Jess!
Do we have a list of things worth checking before merging PR? Alternatively is there an extended CI run we can run?

In regards to x86 - we don't have it in the list of downloads for now, is there any other way we can extract it?
Alternatively, if we want to cover the architectures that we might potentially download are there any other architectures that we should add?

Reply:
no, unfortunately we haven' set up CI for that yet. You will have to manually start a local and a cloud deployment and test all platforms that way

x86: you can only download the via spatial package retrieve, I believe. I think these are all the architectures that we need to test. We only support linux for x64, windows is the only one with x86 and 64